src/main.ts

```diff
@@ -1,13 +1,51 @@
 import { enableProdMode } from "@angular/core";
 import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
-
+import { Amplify } from "aws-amplify";
 import { AppModule } from "./app/app.module";
-import { environment } from "./environments/environment";
+import { environment as defaultEnvironment } from "./environments/environment";
+
+(async () => {
+  const envApiResponse = await fetch("/api/environment");
+  const envName = await envApiResponse.json();
+
+  const response = await fetch("./assets/environments/environment.json");
```

Comment:
Could we concat the `envName` to get values from the standard `environment.<env-name>.ts` files at all? Or is that more effort than its worth?